### PR TITLE
Add token Newscrypto

### PR DESCRIPTION
**Pull request by madcake**

### CODE REVIEW

File: services/markets/coinmarketcap/mapping.go [hunk 1]
```diff
@@ -14199,5 +14199,11 @@ const Mapping = `[
         "type": "token",
         "token_id": "0xa184088a740c695E156F91f5cC086a06bb78b827",
         "id": 8387
+    },
+    {
+        "coin": 60,
+        "type": "token",
+        "token_id": "0x968f6f898a6df937fc1859b323ac2f14643e3fed",
```
Comment (vikmeup) [inline services/markets/coinmarketcap/mapping.go:14206]:
```suggestion
        "token_id": "0x968F6f898a6Df937fC1859b323aC2F14643e3fED",
```

File: services/markets/coinmarketcap/mapping.go [hunk 2]
```diff
@@ -14199,5 +14199,11 @@ const Mapping = `[
         "type": "token",
         "token_id": "0xa184088a740c695E156F91f5cC086a06bb78b827",
         "id": 8387
+    },
+    {
+        "coin": 60,
+        "type": "token",
```
Comment (vikmeup) [inline services/markets/coinmarketcap/mapping.go:14205]: can you create an issue to refactor this part to map as for later:
```
{
    "id: 123,
    "asset_id": "c60_t0x123`
}
```

not urgent, but will make things simpler for the future updates

**Review by hewigovens — APPROVED** [review]
Didn't know it's manually done

**Comment by vikmeup** [comment]
> Didn't know it's manually done

CMC and CG only support 1 asset = 1 token. 

In the case of USDT there is 1 USDT and 4 different assets on different blockchains. So we need to map CMC ID => tron USDT, ETH USDT and so now

**Comment by vikmeup** [comment]
Also both CMC and CG might not even provide correct information from the API then we need to map them too, but currently we use CMC mapping as it covers majority of the assets

**Comment by hewigovens** [comment]
> Also both CMC and CG might not even provide correct information from the API then we need to map them too, but currently we use CMC mapping as it covers majority of the assets

Ask them to add this mapping id to asset repo?

**Comment by vikmeup** [comment]
> > Also both CMC and CG might not even provide correct information from the API then we need to map them too, but currently we use CMC mapping as it covers majority of the assets
> 
> Ask them to add this mapping id to asset repo?

Who to ask?

**Comment by hewigovens** [comment]
Token like Newscrypto, when they added its info to assets repo

**Comment by vikmeup** [comment]
> Token like Newscrypto, when they added its info to assets repo

https://developer.trustwallet.com/add_new_asset#collections, it's part of requirements. It's their responsibility to do it. But we also do this whenever we find missing info

File: services/markets/coinmarketcap/mapping.go [hunk 3]
```diff
@@ -9689,6 +9689,12 @@ const Mapping = `[
         "token_id": "0xade7B5f4a421d81DDaD8Ce86f77A0EfE8921E9CC",
         "id": 4885
     },
+    {
+        "coin": 60,
+        "type": "token",
+        "token_id": "0x968F6f898a6Df937fC1859b323aC2F14643e3fED",
+        "id": 4890
```
Comment (vikmeup) [inline services/markets/coinmarketcap/mapping.go:9696]: Use this `id` to put into correct order

**Pull request merged**